pulumi/src/statefulService.ts

```diff
@@ -335,9 +315,9 @@ export async function deployStatefulService(
 
     const additionalRootDomainName = process.env.ADDITIONAL_ROOT_DOMAIN_NAME
 
-    const match = (service: string, prefix?: string) => {
+    const match = (service: Service, prefix?: string) => {
```

Comment:
did you want to clean up the domain name stuff while you're at it, or follow up pr?

Reply:
preferably the first option